.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Code Question
+about: Ask a question about the code
+title: ''
+labels: 'Type: Question'
+assignees: ''
+
+---
+
+<!--
+Thank you for taking an interest in the OpenZFS codebase.
+
+Please be aware that most questions are preferably asked in the mailinglist first.
+This form is primarily meant for ask questions about the code itself.
+
+Please also check our issue tracker before opening a new question.
+Filling out the following template will help other contributors better understand your proposed feature.
```

Comment:
I think you meant to remove this line, or better yet adapt the "proposed feature" bit at the end.

Reply:
Ohh shoot, I missed that one, thanks!